allureConfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"allure": {
+		"directory": "C:/Users/AlexeyKravchenko/source/repos/allure-results",
```

Comment:
It will be better to remove absolute path because this config will not work on other machines. Just leave allure-results value here